docs/attributes-registry/nodejs.md

```diff
@@ -0,0 +1,15 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# NodeJS
+
+## NodeJS Attributes
+
+Describes Node.js Runtime related attributes.
```

Comment:
are those always only runtime attributes? I wonder if registry as agnostic entity should have `runtime` word
```suggestion
Describes Node.js related attributes.
```

Reply:
good point, removed!